packages/apps/src/app/xcm/App.ts

```diff
@@ -81,6 +81,7 @@ export class XcmApp extends PoolApp {
   @property({ type: String }) destChain: string = null;
   @property({ type: String }) asset: string = null;
   @property({ type: String }) blacklist: string = null;
+  @property({ type: String }) ss58Prefix: string = null;
```

Comment:
if u use type: Number you don't have to cast to Number down below

Reply:
ok, didn't know